website/source/docs/configuration/index.html.md

```diff
@@ -74,8 +74,8 @@ to specify where the configuration is.
 
     Disabling `mlock` is not recommended unless the systems running Vault only
     use encrypted swap or do not use swap at all. Vault only supports memory
-    locking on UNIX-like systems (Linux, FreeBSD, Darwin, etc). Non-UNIX like
-    systems (e.g. Windows, NaCL, Android) lack the primitives to keep a
+    locking on UNIX-like systems that support the mlock() syscall (Linux, FreeBSD, etc). 
+    Non-UNIX like systems (e.g. Windows, NaCL, Android) lack the primitives to keep a
```

Comment:
While you're at it can you fix the grammar -- s/Non-UNIX like/Non UNIX-like/ ? :-D